src/Controller/AppController.php

```diff
@@ -43,6 +43,10 @@ public function initialize()
 
         $this->loadComponent('RequestHandler');
         $this->loadComponent('Flash');
+        //Recommended security enhancements for your application
```

Comment:
In here it is different: https://github.com/cakephp/app/blob/master/config/bootstrap.php#L202-L205

Maybe /* Recommended security enhancements for your application */ ?